python/tests/unit/arcticdb/version_store/test_normalization.py

```diff
@@ -477,3 +477,16 @@ def test_no_inplace_index_array_modification(lmdb_version_store, sym, datetime64
     lmdb_version_store.write(sym, pandas_container)
     assert pandas_container.index is original_index_array
     assert pandas_container.index.dtype == datetime64_dtype
+
+
+def test_pyarrow_error(lmdb_version_store):
+    _ = pytest.importorskip("pyarrow")
```

Comment:
We expect `pyarrow` to be installed don't we? So when would we ever want to skip this silently?

Reply:
`pyarrow` is only installed for tests but is not a run dependencies. Hence, if someone were to test ArcticDB independently, this test would fail at the creation of the structures.

`pyarrow` is present in `environment_unix.yaml` for testing purposes, this file needs to be updated and commented accordingly.

Reply:
If someone tests ArcticDB independently, they should install our test deps

Reply:
I removed this line.